### ISSUE TITLE
[BUG] Cannot read properties of undefined (reading 'replace')

### ISSUE THREAD

**Issue by JimmyZJX** [Vehmloewff/custom-format#13]
I got the following error after upgrading to the latest version. I'm using the Windows client and remote-SSH to a Linux machine.
```
  ERR TypeError: Cannot read properties of undefined (reading 'replace')
	at module.exports [as default] (vscode-file://vscode-app/.../.vscode-server/extensions/vehmloewff.custom-format-1.0.3/node_modules/strip-color/index.js:11:14)
	at Object.<anonymous> (vscode-file://vscode-app/.../.vscode-server/extensions/vehmloewff.custom-format-1.0.3/out/run-command.js:60:65)
	at Generator.throw (<anonymous>)
	at rejected (vscode-file://vscode-app/.../.vscode-server/extensions/vehmloewff.custom-format-1.0.3/out/run-command.js:6:65)
```

**Comment by Indy-rbo** [comment]
@Vehmloewff If you think you need ANSI color stripping I'd suggest using https://github.com/chalk/strip-ansi instead of the `strip-color` package which is more than 8 years old and [only uses a simple regex](https://github.com/jonschlinkert/strip-color/blob/master/index.js). Might improve your experience and may already resolve this issue

**Comment by Vehmloewff** [comment]
Ah, good idea. Will do.

**Comment by Vehmloewff** [comment]
Should've been fixed by 6486f9815530c6cf4559005ab5ea96760bbef262 which is now published under `1.0.5`
